core/src/main/java/io/questdb/log/HttpLogAlertBuilder.java

```diff
@@ -0,0 +1,158 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.log;
+
+import io.questdb.std.*;
+
+
+public class HttpLogAlertBuilder extends LogRecordSink {
+
+    private static final String QDB_VERSION = "7.71.1";
```

Comment:
perhaps clarify what this is - user agent version

Reply:
what is it? that version was in the original source code. I isolated it and named it that, I suppose it should be questdb's runtime version.

Reply:
Usage of the version is not clear from the name of the constant. "version" is too generic, which prompts unnecessary search for usages just to find out that "ah, it is used in user agent http header"